README.md

```diff
@@ -38,8 +38,17 @@ If you want to create a standalone version of Awair-uploader, run the following
 npm run make
 ```
 
+## Installation (Debian based Distros)
+Firstoff you need to install node 12:
+```sh
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+npm install
+npm start
+```
+i couldn get the packaging thingy to works, so as of now you just can compile and run the binary like described above.
```

Comment:
There's no packaging in place for Debian, so for now you can compile and run the binary from source.

Reply:
exactly.